specs/integration/search/3 - CreateDatasetFeature.js

```diff
@@ -0,0 +1,20 @@
+description: "Create a dataset.",
+
+scenario: [
+    CreateDatasetWidget.contribute(),
+    CreateDatasetWidget.publishDataset(),
+    CreateDatasetWidget.next(),
+    CreateDatasetWidget.fill(datasetName, datasetDescription),
```

Comment:
> What you're doing here is already excellent, it's very clear and tidy, it is a great initial investment. When you add tests on a per-feature basis though, it might be worth adding a few intermediary assertions to catch errors in the navigation path  :)